.github/workflows/publish-docs.yaml

```diff
@@ -27,7 +27,7 @@ on:
   workflow_dispatch:
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.ref != 'refs/heads/master' && github.ref || github.run_id }}
+  group: ${{ github.workflow }}
```

Comment:
This is probably too loose, A build of a tagged release could be skipped if merges are committed to master after a tag has been made, if the publish workflow for the tagged commit has not finished the publish action in atomvm_www...